discovery-provider/es-indexer/src/indexers/BaseIndexer.ts

```diff
@@ -44,6 +44,12 @@ export abstract class BaseIndexer<RowType> {
     }
   }
 
+  async refreshIndex() {
+    const { es, logger, indexName } = this
+    logger.info('refreshing index: ' + indexName)
+    await es.indices.refresh({ index: indexName })
```

Comment:
it's a few ms each index so probably ok without logging. it was added here since we extended the refresh_interval to 5s. an edge case would be when the indexer first catches up we want to refresh before making it available otherwise there could be no results for a few seconds. it's also necessary for test cases since we don't want to wait 5s.